tests/sles4sap/sap_suse_cluster_connector.pm

```diff
@@ -59,9 +60,10 @@ sub run {
 
     # Test Maintenance Mode
     foreach my $mod (1, 0) {
-        exec_conn_cmd(binary => $binary, cmd => "smm --sid $instance_sid --ino $instance_id --mod $mod", log_file => $log_file);
+        my $retval = exec_conn_cmd(binary => $binary, cmd => "smm --sid $instance_sid --ino $instance_id --mod $mod", log_file => $log_file);
+        die "Commad 'smm' failed and returns $retval" if ($retval == 2);
```

Comment:
2 only is problematic? How ssm behave in case of error? Maybe you could add a comment explaining why exactly only 2 is problematic and not all the others non zero

Reply:
Done. Yes only 2 is problematic.